indico/web/client/js/react/components/DatePicker.jsx

```diff
@@ -47,7 +49,10 @@ export default function DatePicker({
       <button type="button" disabled={inputProps.disabled}>
         <Translate as="span">Open a calendar</Translate>
       </button>
-      <ind-calendar>
+      <ind-calendar
```

Comment:
I found an interesting bug w/ the min/max logic: When you use alt+down to open a datepicker's calendar which has a minimum value set that prevents selecting the first shown day (I think), then using the tab key will cycle through the navigation buttons and dropdowns, but after the last button pressing tab again closes the calendar instead of jumping to the first available day.

So with `min="2024-05-26"` it works fine, but with `min="2024-05-27"` it breaks.

Reply:
Will check on Monday